generators/server/index.js

```diff
@@ -133,7 +133,8 @@ module.exports = class extends ServerGenerator {
                 writeFiles.call(this);
             }
         };
-        return Object.assign(phaseFromJHipster, customPhaseSteps);
+        // The writing phase is completely overridden
```

Comment:
Define the writing phase in this way:
``

    get writing() {
        // The writing phase is being overriden so that we can write our own templates as well.
        // If the templates doesnt need to be overrriden then just return `super._writing()` here
        const phaseFromJHipster = super._writing();
        const customPhaseSteps = writeFiles();

        return Object.assign(phaseFromJHipster, customPhaseSteps);
    }
``

Reply:
Sure. Thanks for the hint, changes applied.